### ISSUE TITLE
bitvec panic on specific trace file

### ISSUE THREAD

**Issue by tmplt** [rtic-scope/itm-decode#2]
With the attached trace file, `itm-decode` panics with:
```
stack backtrace:
   0: rust_begin_unwind
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/panicking.rs:92:14
   2: core::panicking::panic
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/panicking.rs:50:5
   3: <u8 as core::ops::bit::ShlAssign>::shl_assign
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/bit.rs:913:17
   4: <bitvec::slice::BitSlice<bitvec::order::Lsb0,T> as bitvec::field::BitField>::load_le
             at /home/tmplt/.cargo/registry/src/github.com-1ecc6299db9ec823/bitvec-0.19.5/src/field.rs:349:6
   5: bitvec::field::BitField::load
             at /home/tmplt/.cargo/registry/src/github.com-1ecc6299db9ec823/bitvec-0.19.5/src/field.rs:132:10
   6: itm_decode::Decoder::pull
             at ./src/lib.rs:373:29
   7: itm_decode::main
             at ./src/bin/itm-decode.rs:36:23
   8: core::ops::function::FnOnce::call_once
             at /rustc/cb75ad5db02783e8b0222fee363c5f63f7e2cf5b/library/core/src/ops/function.rs:227:5
```

[itm.bin.zip](https://github.com/tmplt/itm-decode/files/6086173/itm.bin.zip)

**Comment by tmplt** [comment]
This appears to occur with any trace file. Two packets are successfully decoded, then we panic on `bitvec::field::BitField::load`.

**Comment by tmplt** [comment]
Unable to repro today.

**Comment by tmplt** [comment]
`bitvec::field::BitField::load` is no longer used. Presumed to be an edge-case issue with 0.19.4, that is fixed in 0.21.1.
